WebRoot/js/ajax/dwt/widgets/DwtCheckbox.js

```diff
@@ -90,13 +90,26 @@ DwtCheckbox.TEXT_RIGHT			= "right";
  */
 DwtCheckbox.DEFAULT_POSITION	= DwtCheckbox.TEXT_RIGHT;
 
+/**
+ * Icons for custom checkbox style
+ */
+DwtCheckbox.CUSTOM_STYLE_ICONS	= {
+	CheckBox: "CheckboxUnchecked",
+	CheckBoxChecked: "CheckboxChecked"
+};
+
```

Comment:
why we updating the checkbox styles in 790? I believe similar changes have been done in previous pull requests.